src/Board.cpp

```diff
@@ -64,7 +150,59 @@ Lines Board::getCluesRows() const
     return cluesRows_;
 }
 
+bool Board::isSolved()
+{
+    for(auto line : board_)
```

Comment:
extract this loop to a function named checkIfAnyFieldIsUnknown

Reply:
done